scripts/machine_translation/README.md

```diff
@@ -1,170 +1,328 @@
 # Machine Translation
 
+To run the scripts, you are recommended to install [tensorboardX](https://github.com/lanpa/tensorboardX).
+
+```
+python3 -m pip install tensorboardX
+```
+
+Also, to install horovod, you can try out the following command:
+
+```
+HOROVOD_GPU_OPERATIONS=NCCL HOROVOD_WITHOUT_GLOO=1 HOROVOD_WITH_MPI=1 HOROVOD_WITH_MXNET=1 HOROVOD_WITHOUT_TENSORFLOW=1 pip install --no-cache-dir horovod
+```
+
 ## Train a Transformer from scratch
 First, use the script described in [datasets/machine_translation](../datasets/machine_translation) 
 to generate the dataset. Then, run `train_transformer.py` to train the model. 
 
 In the following, we give the training script for WMT2014 EN-DE task with yttm tokenizer. 
 You may first run the following command in [datasets/machine_translation](../datasets/machine_translation).
 ```bash
-bash ../datasets/machine_translation/wmt2014_ende_base.sh yttm # (For transformer_base config)
-bash ../datasets/machine_translation/wmt2014_ende.sh yttm # (For transformer_wmt_en_de_big config)
+bash ../datasets/machine_translation/wmt2014_ende.sh yttm
 ```
 
 Then, you can run the experiment.
-For "transformer_base" configuration
+
+### Transformer Base
+
+For "transformer_base" configuration, we provide the script that uses horovod + amp for training.
 
 ```bash
 SUBWORD_ALGO=yttm
 SRC=en
 TGT=de
-python3 train_transformer.py \
+lr=0.0016
+num_accumulated=16
+max_num_tokens=4096
+epochs=60
```

Comment:
all uppercase to be consistent with the other parameterized arguments?

Reply:
Both are fine for me.